testpilot/frontend/templates/testpilot/frontend/index.html

```diff
@@ -11,6 +10,26 @@
     <meta name="viewport" content="width=device-width">
     <link rel="localization" href="{{ settings.STATIC_URL }}locales/{locale}/app.l20n">
 
+    {% set meta_title = "Firefox Test Pilot" %}
+    {% set meta_description = "Test new Features. Give us feedback. Help build Firefox." %}
+    {% set meta_url = request.build_absolute_uri() %}
+
```

Comment:
Do we need/want to add something like `<meta charset="UTF-8">` here (or at the very top of `<head>`)?